src/main/java/io/stargate/sgv2/jsonapi/service/schema/collections/CollectionRerankDef.java

```diff
@@ -54,32 +54,39 @@ public class CollectionRerankDef {
    *
    * <p>This constructor is annotated with {@link JsonCreator} to enable Jackson deserialization.
    * {@link JsonProperty} annotations on parameters allow Jackson to map JSON properties to
-   * constructor parameters during deserialization.
+   * constructor parameters during deserialization. This constructor is public primarily to support
```

Comment:
Note: constructors do NOT have to be `public` for Jackson -- any visibility is fine, even `private`, as long as `@JsonCreator` annotation is used to mark it for use.